vcd/resource_vcd_vm_insert_eject_media_test.go

```diff
@@ -0,0 +1,220 @@
+package vcd
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"strings"
+	"testing"
+)
+
+var TestAccVcdMediaInsert = "TestAccVcdMediaInsertBasic"
+var vappNameForInsert = "TestAccVcdVAppForInsert"
+var vmNameForInsert = "TestAccVcdVAppVmForInsert"
+var TestAccVcdCatalogMediaForInsert = "TestAccVcdCatalogMediaBasicForInsert"
+var TestAccVcdCatalogMediaDescriptionForInsert = "TestAccVcdCatalogMediaBasicDescriptionForInsert"
+var TestAccVcdVAppVmNetForInsert = "TestAccVcdVAppVmNetForInsert"
+
+func TestAccVcdMediaInsertBasic(t *testing.T) {
+	//t.Skip(acceptanceTestsSkipped)
+	var params = StringMap{
+		"Org":              testConfig.VCD.Org,
+		"Vdc":              testConfig.VCD.Vdc,
+		"EdgeGateway":      testConfig.Networking.EdgeGateway,
+		"Catalog":          testConfig.VCD.Catalog.Name,
+		"CatalogItem":      testConfig.VCD.Catalog.Catalogitem,
+		"VappName":         vappNameForInsert,
+		"VmName":           vmNameForInsert,
+		"CatalogMediaName": TestAccVcdCatalogMediaForInsert,
+		"Description":      TestAccVcdCatalogMediaDescriptionForInsert,
+		"MediaPath":        testConfig.Media.MediaPath,
+		"UploadPieceSize":  testConfig.Media.UploadPieceSize,
+		"UploadProgress":   testConfig.Media.UploadProgress,
+		"InsertMediaName":  TestAccVcdMediaInsert,
+		"NetworkName":      TestAccVcdVAppVmNetForInsert,
+	}
+
+	if vcdShortTest {
+		//	t.Skip(acceptanceTestsSkipped)
```

Comment:
Please keep the skip notice. If we run the test isolated without `TF_ACC`, we need to know that it was skipped for a reason.

Reply:
fixed